Kickstarter-iOS/Views/LoadingButton.swift

```diff
@@ -0,0 +1,115 @@
+import Library
+import Prelude
+import UIKit
+
+final class LoadingButton: UIButton {
+  // MARK: - Properties
+
+  private lazy var activityIndicator: UIActivityIndicatorView = {
+    UIActivityIndicatorView(style: .white)
```

Comment:
Style is something that could potentially be injected in the initializer in the future but for now I think this style will suffice